server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/AliasRoutesTest.java

```diff
@@ -178,6 +185,26 @@ void putShouldDetectConflicts() {
                 .containsEntry("message", "Creation of redirection of alice-alias@b.com to alias:alice@b.com would lead to a loop, operation not performed");
         }
 
+        @Test
+        void putShouldDetectConflictsWithGroups() throws Exception {
+            memoryRecipientRewriteTable.addGroupMapping(MappingSource.fromUser(Username.of(ALICE_ALIAS)), BOB);
+
+            Map<String, Object> errors = when()
+                    .put(ALICE + SEPARATOR + "sources" + SEPARATOR + ALICE_ALIAS)
+                .then()
+                    .contentType(ContentType.JSON)
+                    .statusCode(HttpStatus.BAD_REQUEST_400)
+                    .extract()
+                    .body()
+                    .jsonPath()
+                    .getMap(".");
+
+            assertThat(errors)
+                .containsEntry("statusCode", HttpStatus.BAD_REQUEST_400)
+                .containsEntry("type", "InvalidArgument")
+                .containsEntry("message", "'alice-alias@b.com' already have associated mappings: group:bob@b.com");
+        }
+
```

Comment:
Where are the tests for:

```
GIVEN a user marketing@linagora.com
WHEN I create the alias marketing@linagora.com
THEN it fails
```

```
GIVEN an alias marketing@linagora.com
WHEN I create the alias marketing@linagora.com
THEN it fails
```
?

Reply:
```
GIVEN an alias marketing@linagora.com
WHEN I create the alias marketing@linagora.com
THEN it fails
```

Invalid: a user can have several aliases...

```
GIVEN a user marketing@linagora.com
WHEN I create the alias marketing@linagora.com
THEN it fails
```

Test case already covered by `putExistingUserAsAliasSourceShouldNotBePossible`